apis/core/v1alpha2/flagsourceconfiguration_types.go

```diff
@@ -63,6 +63,15 @@ type FlagSourceConfigurationSpec struct {
 	// LogFormat allows for the sidecar log format to be overridden, defaults to 'json'
 	// +optional
 	LogFormat string `json:"logFormat"`
+
```

Comment:
Also here, what is the reason why there are changes in v1alpha2 API version? AFAIK once the API version is released, no changes should be made to this resource definition. If there are changes required, the new version should be introduced.

Reply:
ive reverted these changes